packages/alfa-rules/src/sia-r75/rule.ts

```diff
@@ -46,13 +47,32 @@ export default Rule.Atomic.of<Page, Element>({
       },
 
       expectations(target) {
-        const { value: fontSize } = Style.from(target, device).computed(
-          "font-size"
-        );
+        const texts = target
+          .descendants({
+            flattened: true,
+            nested: true,
+          })
+          .filter(isText)
+          .map((text) => text.parent().filter(isElement))
+          .filter((option) => option.isSome())
+          .map((option) => option.get())
```

Comment:
This sequence of operations can be replaced by `Sequence#collect()`:

```suggestion
          .collect((text) => text.parent().filter(isElement))
```